src/OrchardCore/OrchardCore.DisplayManagement/IDisplayManager.cs

```diff
@@ -6,7 +6,15 @@ namespace OrchardCore.DisplayManagement
     public interface IDisplayManager<TModel>
     {
         Task<IShape> BuildDisplayAsync(TModel model, IUpdateModel updater, string displayType = "", string groupId = "");
-        Task<IShape> BuildEditorAsync(TModel model, IUpdateModel updater, bool isNew, string groupId = "", string htmlPrefix = "");
-        Task<IShape> UpdateEditorAsync(TModel model, IUpdateModel updater, bool isNew, string groupId = "", string htmlPrefix = "");
+
+        Task<IShape> BuildEditorAsync(TModel model, IUpdateModel updater, bool isNew, string groupId = "")
```

Comment:
Can we make these obsolete such that old compiled libraries use the obsolete methods, but new code knows it should use the new methods.

Reply:
No need to make it obsolete as its just an overloading

Reply:
Also you can't `obsolete` exiting and add new method with default param as you can't compile the code as compiler complains about ambiguous error

Reply:
Seems to build with this:

```c#
using System;
using System.Threading.Tasks;
using OrchardCore.DisplayManagement.ModelBinding;

namespace OrchardCore.DisplayManagement
{
    public interface IDisplayManager<TModel>
    {
        Task<IShape> BuildDisplayAsync(TModel model, IUpdateModel updater, string displayType = "", string groupId = "");
        Task<IShape> BuildEditorAsync(TModel model, IUpdateModel updater, bool isNew, string groupId, string htmlPrefix);
        Task<IShape> UpdateEditorAsync(TModel model, IUpdateModel updater, bool isNew, string groupId, string htmlPrefix);

        [Obsolete]
        Task<IShape> BuildEditorAsync(TModel model, IUpdateModel updater, bool isNew, string groupId = "")
            => BuildEditorAsync(model, updater, isNew, groupId, "");

        [Obsolete]
        Task<IShape> UpdateEditorAsync(TModel model, IUpdateModel updater, bool isNew, string groupId = "")
            => UpdateEditorAsync(model, updater, isNew, groupId, "");
    }
}

```

There are warning that can be fixed by using the new methods explicitly.

Reply:
@MikeAlhayek which if them you added in `1.4`?

Reply:
The once’s with the prefix

Reply:
I see, I though the obsolete once

Reply:
@sebastienros agree, updated as suggested